packages/react-components/src/components/MIMap/MIMap.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import MapboxMap from './MapboxMap/MapboxMap';
+import GoogleMapsMap from './GoogleMapsMap/GoogleMapsMap';
+
+const mapTypes = {
+    Google: 'google',
+    Mapbox: 'mapbox'
+};
+
+MIMap.propTypes = {
+    apiKey: PropTypes.string.isRequired,
+    gmApiKey: PropTypes.string,
+    mapboxAccessToken: PropTypes.string,
+    center: PropTypes.object,
+    zoom: PropTypes.number
+}
+
+function MIMap({ apiKey, gmApiKey, mapboxAccessToken, center, zoom }) {
```

Comment:
Should we add some definitions to the function and all the parameters here?
Same for the Google and Mapbox maps...